src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Customer.xml

```diff
@@ -78,6 +78,7 @@
         </itemOperations>
 
         <property name="id" identifier="true" writable="false" />
+        <property name="resetPasswordToken" readable="false" />
```

Comment:
do we need to expose this data in response? customer should get this data from email (I guess)

Reply:
unnecessary change, as at first iteration it called resource of customer. I will remove it now